<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysState</class>
 <widget class="QWidget" name="SysState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbR1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R6</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R7</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R9</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R8</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R16</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR17">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R17</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R13</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R12</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R15</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R14</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR18">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R18</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R11</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR25">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R25</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR19">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R19</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR26">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R26</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR22">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R22</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R21</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR24">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R24</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR23">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R23</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR27">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R27</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR20">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R20</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR34">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R34</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR28">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R28</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR35">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R35</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR31">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R31</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR30">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R30</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR33">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R33</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR32">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R32</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR36">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R36</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbR29">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R29</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_15">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_16">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_20">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_21">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_22">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_23">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_24">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_25">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_26">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_27">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_28">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_29">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_30">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_31">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_32">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_33">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_34">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_35">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_36">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbSystate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>系统状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>416</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>671</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>lbR1</zorder>
  <zorder>lbR2</zorder>
  <zorder>lbR3</zorder>
  <zorder>lbR4</zorder>
  <zorder>lbR6</zorder>
  <zorder>lbR5</zorder>
  <zorder>lbR7</zorder>
  <zorder>lbR9</zorder>
  <zorder>lbR8</zorder>
  <zorder>lbR16</zorder>
  <zorder>lbR10</zorder>
  <zorder>lbR17</zorder>
  <zorder>lbR13</zorder>
  <zorder>lbR12</zorder>
  <zorder>lbR15</zorder>
  <zorder>lbR14</zorder>
  <zorder>lbR18</zorder>
  <zorder>lbR11</zorder>
  <zorder>lbR25</zorder>
  <zorder>lbR19</zorder>
  <zorder>lbR26</zorder>
  <zorder>lbR22</zorder>
  <zorder>lbR21</zorder>
  <zorder>lbR24</zorder>
  <zorder>lbR23</zorder>
  <zorder>lbR27</zorder>
  <zorder>lbR20</zorder>
  <zorder>lbR34</zorder>
  <zorder>lbR28</zorder>
  <zorder>lbR35</zorder>
  <zorder>lbR31</zorder>
  <zorder>lbR30</zorder>
  <zorder>lbR33</zorder>
  <zorder>lbR32</zorder>
  <zorder>lbR36</zorder>
  <zorder>lbR29</zorder>
  <zorder>lineEdit_1</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>lineEdit_8</zorder>
  <zorder>lineEdit_9</zorder>
  <zorder>lineEdit_10</zorder>
  <zorder>lineEdit_11</zorder>
  <zorder>lineEdit_12</zorder>
  <zorder>lineEdit_13</zorder>
  <zorder>lineEdit_14</zorder>
  <zorder>lineEdit_15</zorder>
  <zorder>lineEdit_16</zorder>
  <zorder>lineEdit_17</zorder>
  <zorder>lineEdit_18</zorder>
  <zorder>lineEdit_19</zorder>
  <zorder>lineEdit_20</zorder>
  <zorder>lineEdit_21</zorder>
  <zorder>lineEdit_22</zorder>
  <zorder>lineEdit_23</zorder>
  <zorder>lineEdit_24</zorder>
  <zorder>lineEdit_25</zorder>
  <zorder>lineEdit_26</zorder>
  <zorder>lineEdit_27</zorder>
  <zorder>lineEdit_28</zorder>
  <zorder>lineEdit_29</zorder>
  <zorder>lineEdit_30</zorder>
  <zorder>lineEdit_31</zorder>
  <zorder>lineEdit_32</zorder>
  <zorder>lineEdit_33</zorder>
  <zorder>lineEdit_34</zorder>
  <zorder>lineEdit_35</zorder>
  <zorder>lineEdit_36</zorder>
  <zorder>lbSystate</zorder>
  <zorder>pbExit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
